Folding In Sandbox/install_folding_sandbox_on_host.ps1

```diff
@@ -3,8 +3,14 @@ $ProgressPreference = 'SilentlyContinue' #Progress bar makes things way slower
 
 #Check Sandbox is enabled, only thing that requires administrator permissions
 Write-Output 'Checking that Windows Sandbox is installed...'
-If((Get-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -Online).length -ne 1) {
-	Enable-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -All -Online
+#Check if virtualization is enbaled in BIOS
+if((Get-WmiObject Win32_ComputerSystem).HypervisorPresent -eq $true){
+    If((Get-WindowsOptionalFeature -FeatureName 'Containers-DisposableClientVM' -Online).length -eq 1) {
```

Comment:
I think this should be '-ne 1' correct? We want to enable Sandbox if it isn't already. This may have just been due to a merge conflict.

Reply:
My Edit was adding the check in line 7 to check for BIOS settings , but that's another good point . I think even if it's disabled value 1 will be returned , we should check for the status value not the length

Reply:
Okay that's fine! I'll change that in another PR.